apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -109,8 +110,10 @@ const IntegrationsList = ({ data }: { data: inferQueryOutput<"viewer.integration
                 installed
               />
             </div>
-          }
-        />
+          }>
+          {/* TODO: @hariom move this into /app-store/packages GeneralSettings */}
+          <WeatherAppSettings />
```

Comment:
we can't merge this yet, before this is abstracted and moved into packages/app-store since this component now would be shown to any app, not only "weather in calendar"